src/deps.ts

```diff
@@ -52,6 +92,9 @@ export let Snappy:
 try {
   // Ensure you always wrap an optional require in the try block NODE-3199
   Snappy = require('snappy');
+  try {
+    (Snappy as any)[PKG_VERSION] = require('snappy/package.json').version;
```

Comment:
why not convert it to number here, so that we don't have to do the independent parsing logic up  in the other file? if the idea is to future proof it, you could parse the major/minor/patch right here and store in an object, but that's probably overkill, it's not like we're using it all over the repo, so parsing it directly to major for now is probably fine